doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -0,0 +1,574 @@
+
+   The :guilabel:`Activate SIM Card` window appears.
+
+      .. figure:: /images/nrfcloud_activating_sim.png
+         :alt: nRF Cloud - Activating SIM Card
+
+         nRF Cloud - Activating SIM Card
+
+#. In the :guilabel:`SIM ICCID` field, enter the ICCID code that you wrote down.
+#. In the :guilabel:`PUK` field, enter the PUK code that you wrote down.
+#. Read Terms and Privacy Policy. Then select the box next to them.
+#. Click :guilabel:`Activate SIM`.
+#. The :guilabel:`Activate SIM Card` window appears.
+#. Enter your information in the :guilabel:`Activate SIM Card` window. Then, click :guilabel:`Submit` .
+
+   The :guilabel:`SIM Activated successfully` message appears.
+
+Connecting to nRF Cloud
+***********************
+
+To start using nRF Cloud, you need to associate the Thingy:91 to your user account.
+
+To connect the Thingy:91 to nRF Cloud and to associate it with your user account,
+complete the following steps:
+
+1. Make sure the Thingy:91 is switched on.
+#. Wait for the Thingy:91 to connect to the LTE network and to nRF Cloud.
+
+   After a few moments, the nRF Cloud user association process starts.
+   This is indicated by white double pulse blinking of the Thingy:91's RGB LED as indicated in :ref:`Operating states <led_indication>`.
+
+#. To complete the user association, navigate to `nRF Cloud`_.
+#. Click the large plus sign in the upper-left corner.
+
+      .. figure:: /images/nrfcloud_plus_sign_callout.png
+         :alt: nRF Cloud
+
+         nRF Cloud
+
+
+   The :guilabel:`Add New` window appears.
+#. In the :guilabel:`Add New` window, click :guilabel:`LTE Device`.
+
+   The :guilabel:`Activate SIM Card` window appears.
+
+      .. figure:: /images/nrfcloud_activating_sim.png
+         :alt: nRF Cloud - Activating SIM Card
+
+         nRF Cloud - Activating SIM Card
+
+#. Skip the :guilabel:`Active SIM Card` section by clicking on :guilabel:`Skip this step`.
+
+   The :guilabel:`Add LTE Device` window appears.
+
+      .. figure:: /images/nrfcloud_add_lte_device.png
+         :alt: Add LTE Device
+
+         Add LTE Device
+
+#. In the :guilabel:`Device ID` field, enter the text ``nrf-`` and after it the Thingy:91 International Mobile (Station) Equipment Identity (IMEI) code.
+
+   The IMEI is the 15 digit code which you can find on a white sticker on the Thingy:91.
+   The Device ID is case sensitive, so make sure all the letters are lower-case.
+
+   .. figure:: /images/thingy91_pin_imei.svg
+      :alt: PIN and IMEI on Thingy:91
+
+      PIN and IMEI on Thingy:91
+
+#. In the :guilabel:`PIN/HWID` field, enter the Personal Identification Number (PIN) which is printed on the white sticker on the Thingy:91.
+#. Click :guilabel:`Add Device`.
+
+   The message :guilabel:`Device added to account. Waiting for it to connect...` appears.
+
+   .. note::
+
+      If you see an error message, check the error code and see `nRF Cloud REST API`_ in nRF Cloud to find out what is causing the error.
```

Comment:
Is this the correct link? I don't really find any information that can help me debug the problem following that link.